src/raft_log.rs

```diff
@@ -434,7 +441,7 @@ impl<T: Storage> RaftLog<T> {
 
     /// Grabs a slice of entries from the raft. Unlike a rust slice pointer, these are
     /// returned by value. The result is truncated to the max_size in bytes.
-    pub fn slice(&self, low: u64, high: u64, max_size: u64) -> Result<Vec<Entry>> {
+    pub fn slice(&self, low: u64, high: u64, max_size: Option<u64>) -> Result<Vec<Entry>> {
```

Comment:
How about make it as `max_size: impl Into<Option<u64>>`?

Reply:
fixed